src/txn/txn_rollback_to_stable.c

```diff
@@ -479,7 +528,7 @@ __rollback_abort_row_reconciled_page(
                  * eviction may pick the already reconciled page to write to disk with newer
                  * updates.
                  */
-                __wt_page_only_modify_set(session, page);
+                __wt_page_modify_set(session, page);
```

Comment:
Thanks - I really like that you added and included debugging here. It's a bit hard to find the actual fix inside the debugging - would it be difficult to split the debugging into a different ticket?

Reply:
Yes, I agree. Removed them as part of the bug ticket and will add them in a new ticket.